Simplenote/Classes/SPNoteListViewController+Extensions.swift

```diff
@@ -622,6 +705,40 @@ extension SPNoteListViewController {
             notesListController.object(at: indexPath) as? Note
         }
     }
+
+    open override func setEditing(_ editing: Bool, animated: Bool) {
+        super.setEditing(editing, animated: animated)
+        refreshEditButtonTitle()
+        
+        tableView.deselectAllRows(inSection: 0, animated: false)
+        updateNavigationBar()
+
+        tableView.allowsMultipleSelection = editing
```

Comment:
I **think** we can drop this? (during initialization it's not explicitly set).

When we're not in edition mode, pressing over any row causes the Editor to get pushed, and we immediately deselect the selected row.